api/envoy/config/core/v3/health_check.proto

```diff
@@ -370,6 +371,10 @@ message HealthCheck {
   // If empty, no event log will be written.
   string event_log_path = 17;
 
+  // Specifies the event logger for health check event. The underlying typed_config should be
+  // :ref:`access logs <arch_overview_access_logs>`.
+  repeated TypedExtensionConfig event_logger = 25;
```

Comment:
The `event_log_path` should be deprecated in favor of the `event_logger`. We should provide implementation of the `event_logger` that writes JSON into a file.